doc/btp_has.txt

```diff
@@ -0,0 +1,107 @@
+HAS Service (ID 15)
+===================
+
+Commands and responses:
+
+	Opcode 0x00 - Error response
+
+	Opcode 0x01 - Read Supported Commands command/response
+
+		Controller Index:	 <controller id>
+		Command parameters:	 <none>
+		Response parameters: <supported commands> (variable)
+
+		Each bit in response is a flag indicating if command with
+		opcode matching bit number is supported. Bit set to 1 means
+		that command is supported. Bit 0 is reserved and shall always
+		be set to 0. If specific bit is not present in response (less
+		than required bytes received) it shall be assumed that command
+		is not supported.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x02: HAS Set Active Preset Index
+
+		Controller Index:	 <controller id>
+		Command parameters:	 Address_Type (1 octet)
+							 Address (6 octets)
```

Comment:
nitpick,  adjust indentation,  in other doc files we typically use tabs (8 spaces wide)